### PR TITLE
15 add the shiny section

### PR DESCRIPTION
**Pull request by lewandt**
I added the corresponding Shiny APP with demographics TEAL module based on the same ADSL example

### CODE REVIEW

File: content/app/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+pre: "<b>1. </b>"
+chapter: true
+description: "Shiny APP"
+---
+
+### Section 3
+
+# Shiny APP examples
+
+This section will contain examples creating APP with demographics output.
```
Comment (rossfarrugia) [inline content/app/_index.en.md:13]: maybe after APP add: (interactive data display) or something like that for people completely new to what this is

File: content/app/teal/_index.Rmd [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## Introduction
+
+This guide will show you how set of the pharmaverse packages can be used to create a a Shiny APP using demographic module end-to-end, using the {admiral} ADSL data as an input.
+
+The eight packages used with a brief description of their purpose are as follows:
+
+* [`{teal.widgets`](https://insightsengineering.github.io/teal.widgets/main/index.html): provides various widgets for teal applications.
+* [`{teal.logger`](https://insightsengineering.github.io/teal.logger/main/index.html): provides a unified setup for generating logs using the logger package.
+* [`{teal.code`](https://insightsengineering.github.io/teal.code/main/index.html):provides tools to store code and an execution environment associated with it.
```
Comment (rossfarrugia) [inline content/app/teal/_index.Rmd:18]: add space after ":"

File: content/app/teal/_index.Rmd [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+* [`{teal.data`](https://insightsengineering.github.io/teal.data/main/index.html): creates the data needed for teal applications.
+* [`{teal.slice`](https://insightsengineering.github.io/teal.slice/main/index.html): used in the development of teal shiny modules
+* [`{teal.transform`](https://insightsengineering.github.io/teal.transform/main/index.html): contains functions and shiny modules for extracting and merging data within the teal framework.
+* [`{teal`](https://insightsengineering.github.io/teal/main/index.html): is a shiny-based interactive exploration framework for analyzing data
+* [`{teal.modules.clinical`](https://insightsengineering.github.io/teal.modules.clinical/main/index.html): contains a set of standard teal modules to be used with CDISC data in order to generate many of the standard outputs used in clinical trials
+
+
+In this exmaple it is important to understand [`{teal.modules.clinical`](https://insightsengineering.github.io/teal.modules.clinical/main/index.html) in order to modify parametrs to the teal module: `tm_t_summary`.
```
Comment (rossfarrugia) [inline content/app/teal/_index.Rmd:26]: parameters

File: content/app/teal/_index.Rmd [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        c("SEX", "RACE")
+      ),
+      useNA = "ifany"
+    )
+  )
+)
+
+```
+
+Now we run the shiny APP. Please note that parmeter options is only used to control rendering of the Shiny APP.
```
Comment (rossfarrugia) [inline content/app/teal/_index.Rmd:114]: parameter option

**Review by rossfarrugia — CHANGES_REQUESTED** [review]
thanks @lewandt - looks good! only some very minor typos spotted

**Review by lewandt — COMMENTED** [review]
I amended all the changes. The code will still not work until the renv is updated, but I am working on it.

**Review by rossfarrugia — APPROVED** [review]

**Comment by rossfarrugia** [comment]
ok thanks! i approved and feel free to merge once renv issue resolved

**Pull request merged**